### DISCUSSION TITLE
How to remove "https://" prefix from omnibox?

### DISCUSSION THREAD

**Discussion by emvaized** [bromite/bromite#2418] [Q&A] [answered]
Is there some flag or settings option to not constantly show "https://" prefix in the omnibox? I feel like I don't need to see this information all the time when browser by default enforces https mode automatically.
![Screenshot_20221030-013457_Bromite](https://user-images.githubusercontent.com/37851576/198854942-9e855e80-0ddc-4db8-a43d-1384aa8ca713.jpg)

**Comment by csagan5** [comment, marked as answer]
It is currently not possible to disable that in Bromite because there is some complex logic in Chromium that decides when to hide it or display it; rather than putting that logic in control in Bromite it is always displayed.

**Reply by emvaized** [reply]
Okay, thanks for answering! I just thought it was one of the applied patches, since in pure Chromium it's not usually visible

**Reply by csagan5** [reply]
Yes, it is added with this patch: https://github.com/bromite/bromite/blob/106.0.5249.72/build/patches/Disable-omission-of-URL-elements-in-Omnibox.patch

If you make your own build you can omit it and achieve the desired effect.

**Reply by emvaized** [reply]
Thanks, I definitely would try it – as I anyway wanted to try creating a new patch for removing the close buttons in the tab switcher.
